rfcs/2021-28-07-motion-design-core-actions.md

```diff
@@ -0,0 +1,176 @@
+1. **Scale:** `length`, `height`, `anchor-point`, `size`.
+2. **Move:** `anchor-points`, `pathway`.
+3. **Fade:** `opacity`.
+4. **Fill:** `colour-tokens`.
+5. **Morph:** `shape-attributes`.
+
+## Atrribute Details
+### 1. **Scale**
+* **Length:** Size on `x-axis`.
+* **Height:** Size on `y-axis`.
+* **Anchor point:** Coordinates of a point in the UI element which define the position of the element; usually it’s the center of the element. Basically a point in which all transformations are manipulated from.
+* **Size:** Percentage(`%`) change in size if aspect ratio is constant.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Zoom** | Where the `aspect-ratio` is the same; don’t squash and stretch UI elements<br>**Ex:** Hovering a card.<br>**Unit:** `%` | <img src="images/motion/core-actions/image4.gif" alt="core-action-scale-zoom" width="200px"> |
+| **Expansion** | Where `length` and `height` changes separately.<br>**Ex:** Expansion of a modal or opening of a dropdown.<br>**Unit:** `px`,`%`,`%y`,`%` | <img src="images/motion/core-actions/image5.gif" alt="core-action-scale-expand" width="200px"> |
+
+### 2. **Move**
+* **Anchor points(`I` & `F`):** Initial & Final anchor-point coordinates; `(x1, y1)(x2, y2)`.
+* **Pathway:** Control points (which decide curve of the line) [path](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths) i.e., how the element moves from `position1` to `position2` representing the pathways; depending on the surroundings or the `z-axis`.
+
+
+### Pathway Types:
+| Type | Description | Example |
+|---|---|---|
+| **Type 1:** <br><br> <img src="images/motion/core-actions/image6.gif" alt="core-action-scale-expand" width="200px"> | **Elements in the same layer (`z-space`)** <br>  Use type 1 pathway, it’s straight-forward and avoids unnecessary distraction.| <img src="images/motion/core-actions/image8.png" alt="core-action-scale-expand" width="200px"> |
+| **Type 2:**  <br><br> <img src="images/motion/core-actions/image17.gif" alt="core-action-scale-expand" width="200px">| **No other element in the same layers** <br> Use type-2 pathway, it makes space for the movement<br>**Ex:** Check out Spotify play icons and top-bar icons.  | <img src="images/motion/core-actions/image7.gif" alt="core-action-scale-zoom" width="200px"> |
+
+### 3. **Fade**
+* **Opacity:** Percentage(`%`) change in the `opacity`.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Fade In** | Opacity value moves from a lower to higher. <br>**Ex:** Appearing of a card on a page.<br>**Unit:** `%` | <img src="images/motion/core-actions/image9.gif" alt="core-action-fade-in" width="200px"> |
+| **Fade Out** | Opacity value moves from a higher to lower.<br>**Ex:** Disappearing of a card from a page.<br>**Unit:** `%` | <img src="images/motion/core-actions/image9.gif" alt="core-action-fade-out" width="200px"> |
```

Comment:
the fade in and and fade out images are same in the example. By mistake?

Reply:
I need to edit this, Image containing both Fade-in and Fade-out. Will add new images.

Reply:
@dubeydushyant Have we edited these?